spec/models/api/trainee_filter_params_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe Api::TraineeFilterParams do
+  describe "since validation" do
+    it "validates since date" do
```

Comment:
I would suggest to add an expectation when the date throws a `Date::Error` according to the code

Reply:
Yes my bad! Nice